typescript/infra/test/warp-configs.test.ts

```diff
@@ -24,6 +24,20 @@ const warpIdsToSkip = [
   'USDT/base-celo-fraxtal-ink-lisk-mode-optimism-soneium-superseed-unichain-worldchain',
 ];
 
+async function getRegistryWithFallback(warpRouteId: string) {
+  const getConfigForBranch = async (branch: string) => {
+    const registry = getRegistry({
+      registryUris: [DEFAULT_GITHUB_REGISTRY],
+      enableProxy: true,
+      logger: rootLogger,
+      branch,
+    });
+    return registry.getWarpDeployConfig(warpRouteId);
+  };
+
+  const mainConfig = await getConfigForBranch('main');
+  return mainConfig ?? getConfigForBranch('main~5');
```

Comment:
is 5 enough?

Reply:
maybe. you like 10 more?

Reply:
(don't have a strong opinion about this)

Reply:
haha let's start with 5 then bump up if we feel it's needd

Reply:
is there a registry PR in-flight we can test this with?

1. merge registry PR that causes builds to fail
2. expect this PR not to fail

Reply:
unfortunately not. but that was my plan to test before merging this

Reply:
![Screenshot 2025-04-28 at 5 45 25 PM](https://github.com/user-attachments/assets/0de2836f-a94f-44e7-95ae-743f48ec3983)

For context, EZETH was updated in the Registry to include worldchain, this PR doesnt worldchain